src/test/java/org/arquillian/container/chameleon/ConfigurationTestCase.java

```diff
@@ -80,4 +80,22 @@ public void shouldFailOnMissingContainerFile() throws Exception {
         configuration.setChameleonTarget("wildfly:8.2.0.Final:managed");
         configuration.validate();
     }
+
+    @Test
+    public void shouldReturnFalseForMissingContainerName() throws Exception {
+        boolean isSupported = ChameleonConfiguration.isSupported("MISSING_TARGET:8.2.0.Final:managed");
+        Assert.assertFalse(isSupported);
```

Comment:
Chameleon has `assertj`dependency defined so I suggest to move these assertions to assertj.